extension/chrome/elements/reply_pubkey_mismatch.ts

```diff
@@ -120,9 +120,9 @@ Catch.try(async () => {
   const determineReplyHeaders = async () => {
     const thread = await Google.gmail.threadGet(acctEmail, threadId, 'full');
     if (thread.messages && thread.messages.length > 0) {
-      const threadMsgIdLast = Google.gmail.findHeader(thread.messages[thread.messages.length - 1], 'Message-ID') || '';
+      const replyMsgIdLast = Google.gmail.findHeader(thread.messages[thread.messages.length - 1], 'Message-ID') || '';
```

Comment:
Here needs improving, I can fix it.

Since we don't always just reply to last message, but to a specific message, we should take care to construct the headers using the message we are replying to, not the last in thread.

Reply:
Actually, this is just reply_pubkey_mismatch which is not a problem. You will probably refactor this away in #2128